server/src/main/java/com/cloud/resourcelimit/ResourceLimitManagerImpl.java

```diff
@@ -521,13 +520,21 @@ public void checkResourceLimit(final Account account, final ResourceType type, l
         }
 
         final Project projectFinal = project;
+
+        // Check account limits. If it's unlimited then don't lock the db rows
```

Comment:
This comment says no more than the code below, we might as well drop it.

Reply:
ok